pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -281,56 +253,25 @@ spec:
       value: somethingfun
     - name: bar
       value: $(params.bar)
-    - name: templatedparam
```

Comment:
Do you know if this functionality (templated param) is only available for PipelineResources? In case it's generally available we should switch this to a regular parameter instead of removing it, or else we'll lose test coverage

Reply:
sorry i might be mistaken here but it felt to me `templatedparam` should be the name of a param?

Reply:
I think Andrea was referring to the `$(inputs.workspace.$(params.rev-param))` usage above. I'm not 100% sure if this is supported elsewhere. cc @jerop do you know?